checker/tests/index/BasicSubsequence.java

```diff
@@ -0,0 +1,44 @@
+import org.checkerframework.checker.index.qual.*;
+
+class BasicSubsequence {
```

Comment:
Need tests for `@HasSubsequence` with invalid flow expressions.

Reply:
This was a good thought, because what I have definitely doesn't handle invalid flow expressions correctly. I'm not sure what the right way to handle it is, though - I'd like to be able to test whether a String is a valid FlowExpression, but I don't know how to do that other than trying to parse it. Maybe that's the right answer?